libbeat/filter/filter.go

```diff
@@ -65,6 +65,11 @@ func (filters *FilterList) Get(index int) FilterRule {
 // Applies a sequence of filtering rules and returns the filtered event
 func (filters *FilterList) Filter(event common.MapStr) common.MapStr {
 
+	// Check if filters are set, just return event if not
+	if filters == nil {
```

Comment:
You are right, not sure why I introduced it. Will remove it and see if everything still works as expected ...